assets/community/bounces.toml

```diff
@@ -9,16 +9,16 @@
 # BOUNCE RULES FILE:
 # -- load the community bounces.toml + iana.toml + local settings
 # kumo.on('init', function()
-#   kumo.configure_local_logs {
-#     log_dir = '/var/log/kumomta',
-#   }
-#   kumo.configure_bounce_classifier {
-#     files = {
-#       '/opt/kumomta/etc/policy/bounces.toml',
-#       '/opt/kumomta/share/community/bounces.toml',
-#        '/opt/kumomta/share/bounce_classifier/iana.toml',
-#     },
-#   }
+# kumo.configure_local_logs {
+#  log_dir = '/var/log/kumomta',
+# }
+# kumo.configure_bounce_classifier {
+#  files = {
+#   '/opt/kumomta/etc/policy/bounces.toml',
+#   '/opt/kumomta/share/community/bounces.toml',
+#    '/opt/kumomta/share/bounce_classifier/iana.toml',
+#  },
+# }
```

Comment:
please revert the changes from this block; they are not functional changes and will make it more difficult to manage merging in the future.